packages/shared-ux/card/no_data_elastic_agent/src/no_data_elastic_agent_card.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useMemo } from 'react';
+import { RedirectAppLinksContainer } from '@kbn/shared-ux-link-redirect-app';
+
+import {
+  NoDataElasticAgentCard as Component,
+  Props as ComponentProps,
+} from './no_data_elastic_agent_card.component';
+import { useServices } from './services';
+
+export type Props = Omit<ComponentProps, 'canAccessFleet'>;
+
+export const NoDataElasticAgentCard = ({
```

Comment:
Please don't kill me for this now 😬 🙏 
The `NoDataCard` component is not actually being used anywhere besides `ElasticAgentCard`  component ([here](
https://github.com/elastic/kibana/pull/134503/files#diff-71d7709c40cda08c59a9bf6539a525a4ca2fb529984762a78ff53196be55bd18L82))

It used to be, but since we've cleaned up some things inside `PageTemplate`, it is not. I've been meaning to clean that up, but didn't consider it high priority. 
How about we remove that component now and put it inside `NoDataElasticAgentCard`? There is no need for it to be a standalone component, and then we could then name *this* component `NoDataCard`. I think name `NoDataElasticAgentCard` exposes too much implementation details anyway.

Reply:
Sure, I can do that.